audiofile/core/io.py

```diff
@@ -21,20 +20,30 @@ def convert_to_wav(
         outfile: str,
         offset: float = 0,
         duration: float = None,
+        bit_depth: int = 16,
+        normalize: bool = False,
+        **kwargs,
 ):
     """Convert any audio/video file to WAV.
 
-    It uses soundfile for converting WAV, FLAC, OGG files,
-    and sox or ffmpeg for converting all other files.
+    It uses soundfile for reading WAV, FLAC, OGG files,
+    and sox or ffmpeg for reading all other files.
     If ``duration`` and/or ``offset`` are specified
     the resulting WAV file
     will be shortened accordingly.
 
+    It then uses :func:`soundfile.write` to write the WAV file,
+    which limits the number of supported channels to 65535.
+
     Args:
         infile: audio/video file name
         outfile: WAV file name
         duration: return only a specified duration in seconds
         offset: start reading at offset in seconds
+        bit_depth: bit depth of written file in bit,
+            can be 8, 16, 24
+        normalize: normalize audio data before writing
+        kwargs: pass on further arguments to :func:`soundfile.write`
```

Comment:
Do we have a use-case / test for this?

Reply:
No, but I thought as we use `soundfile.write()` under the hood it is convenient to still be able to use all of it options.

Reply:
And we have the same for `audiofile.write()`.